lib/reporters.js

```diff
@@ -0,0 +1,114 @@
+import { Transform } from 'node:stream'
+import { fileURLToPath } from 'node:url'
+
+function normalizeFile (file, cwd) {
+  let res = file
+  if (file.startsWith('file://')) {
+    try {
+      res = fileURLToPath(new URL(file))
+    } catch (err) {
+      if (err.code === 'ERR_INVALID_FILE_URL_PATH') {
+        res = fileURLToPath(new URL(file.replace('file:///', 'file://')))
+      }
+    }
```

Comment:
I don't think your assesment is correct here, my understanding is that the URL that failed to be converted to path is `file:///foo/test/add.test.ts`, which indeed does not make sense on Windows. But that's a value constructed in your tests, not one generated by the test runner IIUC.
```suggestion
  res = fileURLToPath(file)
```

Reply:
This was failing with a Windows path.

Reply:
https://github.com/mcollina/borp/pull/13